pixi.toml

```diff
@@ -30,6 +29,75 @@ matplotlib = ">=3.9.2"
 [target.linux-aarch64.dependencies]
 blas = { version = "*", build = "netlib" }
 
+[package]
+name = { workspace = true }
+version = { workspace = true }
+
+[package.build]
+backend = { name = "pixi-build-cmake", version = "0.3.*" }
+
+[package.build.config]
+extra-args = [
+  "-DGENERATE_PYTHON_STUBS=ON",
+  "-DBUILD_TESTING=OFF",
+  "-DBUILD_WITH_COLLISION_SUPPORT=ON",
+  "-DBUILD_WITH_ACCELERATE_SUPPORT=OFF",
+  "-DBUILD_WITH_CASADI_SUPPORT=OFF",
+  "-DBUILD_WITH_AUTODIFF_SUPPORT=OFF",
+  "-DBUILD_WITH_EXTRA_SUPPORT=OFF",
+  "-DBUILD_WITH_OPENMP_SUPPORT=OFF",
```

Comment:
We turn this and casadi off in the pixi-build by default, because these are the defaults in our CMake listfile?

Reply:
Because only want to build a « minimal » pinocchio package. I think collision and parsers are the bare minimum.